python/docs/lis/userguide.rst

```diff
@@ -75,3 +75,85 @@ Read all the curve data by passing each DFSR to :func:`dlisio.lis.curves`:
+
+.. note::
+   The LIS79 specification does not have a precise definition of when to
+   use which record type. Hence it's up to the producers to decide which of the
+   above record types to use.
+
+The records can be accessed through their own methods on
+:class:`dlisio.lis.LogicalFile`. E.g. all the Wellsite Data Records can be read
+with :func:`dlisio.lis.LogicalFile.wellsite_data()`:
+
+.. code-block:: python
+
+    >>> records = f.wellsite_data()
+    >>> print(records)
+    [InformationRecord(type=lis_rectype.wellsite_data, ltell=62)]
+
+
+This particular file only contains a single Wellsite Data Record, but it's not
+uncommon that a file contains multiple Information Records of the same type.
+Let's have a closer look at it's content:
+
+.. code-block:: python
+
+   >>> inforec = records[0]
+   >>> inforec.components()
+   [dlisio.core.component_block(mnem='TYPE', units='    ', component='CONS'),
+    dlisio.core.component_block(mnem='MNEM', units='    ', component='WN  '),
+    dlisio.core.component_block(mnem='STAT', units='    ', component='ALLO'),
+    dlisio.core.component_block(mnem='PUNI', units='    ', component='    '),
+    dlisio.core.component_block(mnem='TUNI', units='    ', component='    '),
+    dlisio.core.component_block(mnem='VALU', units='    ', component='15/9-F-15 '),
+    dlisio.core.component_block(mnem='MNEM', units='    ', component='CN  '),
+    dlisio.core.component_block(mnem='STAT', units='    ', component='ALLO'),
+    dlisio.core.component_block(mnem='PUNI', units='    ', component='    '),
+    dlisio.core.component_block(mnem='TUNI', units='    ', component='    '),
+    dlisio.core.component_block(mnem='VALU', units='    ', component='StatoilHydro')]
+
+As you can see, Information Records are essentially a list of
+:class:`dlisio.core.component_block`. Each Component Block contains a specific
+piece of information, e.g. a parameter.
+
+Tables in Information Records
+.............................
+
+Some Information Records are intended to be read as tables. I.e. each Component
+Block is a entry in a table. This can be checked by calling
+:func:`dlisio.lis.InformationRecord.isstructured()`:
+
+.. code-block:: python
+
+   >>> inforec.isstructured()
+   True
+
+If so, dlisio can create the table for you, in form of a `Numpy Structured Array
+<https://numpy.org/doc/stable/user/basics.rec.html>`_. The mnemonics from the
+Component Blocks are used as field names (column names):
+
+.. code-block:: python
+
+   >>> inforec.table(simple=True)
+   [('WN  ', 'ALLO', '    ', '    ', '15/9-F-15 '),
+    ('CN  ', 'ALLO', '    ', '    ', 'StatoilHydro')]
+
+Setting the argument ``simple=True`` means that only the values from the
+Component Blocks are used in the table. Setting it to ``False``, which is the
+default behavior, means that the entire Component Blocks are put into the
+array. This is useful if you e.g. want to preserve other parts of the Component
+Blocks, such as the units.
```

Comment:
such as () units

(I am bad with when articles should/shouldn't be present, but here "the" sounds too bad to me)